website/docs/r/container_registry_token.html.markdown

```diff
@@ -0,0 +1,84 @@
+---
+subcategory: "Container"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_container_registry_token"
+description: |-
+  Manages an Azure Container Registry token.
+
+---
+
+# azurerm_container_registry_token
+
+Manages an Azure Container Registry token. Tokens are a preview feature only available in Premium SKU Container registries.
+
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "example-resource-group"
+  location = "West Europe"
+}
+
+resource "azurerm_container_registry" "example" {
+  name                     = "example-registry"
+  resource_group_name      = azurerm_resource_group.example.name
+  location                 = azurerm_resource_group.example.location
+  sku                      = "Premium"
+  admin_enabled            = false
+  georeplication_locations = ["East US", "West Europe"]
+}
+
+resource "azurerm_container_registry_scope_map" "example" {
+  name                    = "example-scope-map"
+  container_registry_name = azurerm_container_registry.acr.name
+  resource_group_name     = azurerm_resource_group.rg.name
+  actions = [
+    "repositories/repo1/content/read",
+    "repositories/repo1/content/create"
+  ]
+}
+
+resource "azurerm_container_registry_token" "example" {
+  name                    = "exampletoken"
+  container_registry_name = azurerm_container_registry.acr.name
+  resource_group_name     = azurerm_resource_group.rg.name
+  scope_map_id            = azurerm_container_registry_scope_map.map.id
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+
+* `name` - (Required) Specifies the name of the token. Changing this forces a new resource to be created.
+
+* `resource_group_name` - (Required) The name of the resource group in which to create the Container Registry token. Changing this forces a new resource to be created.
+
+* `container_registry_name` - (Required) The name of the Container Registry. Changing this forces a new resource to be created.
+
+* `scope_map_id` - (Required) The ID of the Container Registry Scope Map associated with the token.
+
+* `status` - (Optional) The status of the Container Registry token.  Defaults to  `enabled`.
```

Comment:
could we just name this enabled and make it a boolean?

Reply:
`azurerm_container_registry_webhook` uses the same syntax so I'll let you decide if we want to keep it the same over all sub-resources of Container Registry or keep it like it is.

Reply:
Let's change it to enabled please, status = enabled was the older way of doing things. enabled = true is far more clear and autocompletes better. If more status are added we can always deprecate this property